docs/en/remote-flow.rst

```diff
@@ -559,7 +572,8 @@ When the Wallet sends a response using ``direct_post.jwt`` to the Relying Party,
 Redirect URI
```

Comment:
Normative details of the response at the Authorization Response endpoint are missing

Reply:
See my suggestion below